Jenkinsfile

```diff
@@ -149,6 +148,8 @@ def executeTests() {
       ${ISOLATED_ITS_ARGUMENT} \
       ${PARALLELIZABLE_ITS_ARGUMENT}
   '''
+  echo "Invoking Maven with parameters test-jdk-${testJavaVersion} and testJavaHome = ${testJavaHome}"
```

Comment:
Seemed useful to log what we're actually passing to Maven.

Reply:
I solved this by using `bash -lex` instead. It will print out the whole maven command that is passed to the shell. It may provide more info, but either works.

Reply:
Jenkins does something similar in the BlueOcean interface, so we actually had it already.  I was just trying to make it a bit more verbose here. :)